### PR TITLE
refactor: remove old onboarding screens #FF

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

Cleanup PR:
- [x] deletes old onboarding code
- [x] renames OnboardingV2 -> OnboardingQuiz 
- [x] remove `AREnableNewOnboarding` ff from admin menu 

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- onboarding old screens cleanup - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/store/config/features.ts [hunk 1]
```diff
@@ -246,7 +246,6 @@ export const features = defineFeatures({
   AREnableNewOnboarding: {
```
Comment (dimatretyak) [inline src/app/store/config/features.ts:246]: **question**: should we remove this feature flag?

File: src/app/Scenes/Onboarding/Onboarding.tsx [hunk 2]
```diff
@@ -111,23 +110,14 @@ export const OnboardingWelcomeScreens = () => {
...
-        {onboardingState === "incomplete" ? (
-          isNewOnboardingEnabled ? (
-            <OnboardingV2 />
-          ) : (
-            <OnboardingPersonalization />
-          )
-        ) : (
-          <OnboardingWelcomeScreens />
-        )}
+        {onboardingState === "incomplete" ? <OnboardingV2 /> : <OnboardingWelcomeScreens />}
```
Comment (dimatretyak) [inline src/app/Scenes/Onboarding/Onboarding.tsx:120]: **question**: what do you think about renaming `OnboardingV2` to `Onboarding`?

**Review by dimatretyak — APPROVED** [review]

File: src/app/Scenes/Onboarding/Onboarding.tsx (hunk 2 above)
Comment (gkartalis) [reply src/app/Scenes/Onboarding/Onboarding.tsx:120]: weird, I did do that, for some reason it wasn't tracked in github 🤯

Comment (gkartalis) [reply src/app/Scenes/Onboarding/Onboarding.tsx]: ookay, now I pushed every change

Comment (dimatretyak) [reply src/app/Scenes/Onboarding/Onboarding.tsx]: thank you!

**Review by dimatretyak — DISMISSED** [review]

**Pull request merged**